### PR TITLE
[DNM] [WIP] Adds simple mob moodlets

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 
:cl: Dax Dupont
add: In life there's forks in the road every so often. Sometimes these lead to riches, sometimes these lead to rooms full of living moodlets who will do unspeakable horrors to you if attacked.
/:cl:

[why]: It's hilarious

- [ ] Make touching the moodlet affect your mood

### CODE REVIEW

**Comment by Qustinnus** [comment]
make them depress you on hit tbf

**Comment by DaxDupont** [comment]
@Qustinnus I don't know how

**Comment by improvedname** [comment]
make them make you happy if you hit them when your a traitor

**Comment by Qustinnus** [comment]
@DaxDupont It would be better to wait for #36344 to be merged so it wont have to be changed later, one that is merged I'll show you how to add a moodlet to it.

**Comment by DaxDupont** [comment]
I'll wait till merge. Tagged it DNM

**Comment by arsserpentarium** [comment]
Make them make you happy when you hit them.
This thing very annoying. So very desirable to hit it.

**Comment by Qustinnus** [comment]
@arsserpentarium youre just mad because i wont add erp

**Comment by arsserpentarium** [comment]
Erp was only excuse for this thing.
Because it is bad game mechanics,which is needed for lewd games to excuse,why fetish content makes you stronger.
But there is nothing to be excused by this thing on TG.
Also it works for games like Call of Ctulchu. Because you need to enforce somehow property of eldritch horrors to drive someone insane. Because barely anyone can draw something horryble enought in computer game for now.
It works for simulation games like DF,sims. But needs to point at fact,that it works for mobs,not for players.So I suggested to make some kind of psychology for mobs. So you could really pet monkey,or scare space carps. It would be great.
Also barely you can excuse something with this face. It is ugly at level of lifeweb target doll. But there are everything ugly in lifeweb. It just such kind of style. I hope, we won't port their style.

**Comment by Armhulen** [comment]
um, lol?

**Comment by Armhulen** [comment]
Also making sentient hud elements as a mob you can get is only slightly less stupid than shamebrero and I hate this pr.

**Comment by DaxDupont** [comment]
@Armhulen good thing there's a separate of server and code.
This is also gonna be used in the insanity thing @Qustinnus was brainstorming off where if you get a bad enough trauma you go fourth wall breaking insane and the hud element literally breaks off.

File: code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm]: timer

File: code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm]: condense the ifs

File: code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm]: update_icon

File: code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm]: update_icon

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm
Comment (DaxDupont) [reply code/modules/mob/living/simple_animal/hostile/retaliate/moodlet.dm]: code\modules\mob\living\simple_animal\hostile\retaliate\moodlet.dm:39:error: update_icon: undefined proc I have no clue what the fuck

**Comment by Armhulen** [comment]
This isn't a democracy

**Comment by DaxDupont** [comment]
This is a dictatorship of the maintainers. 
Behead whoever says the maintainers are violent.

**Comment by Armhulen** [comment]
I don't care how many emojis you can collect on your pr, it's either a gold spawn mob that makes absolutely no sense in its existence or another admins toy that will never be used ((SHAMEBRERO))

Using the "server codebase separation" argument and "the people like it" argument in the same post blows my mind because like I said the only people that matter in getting your pr through are the maintainers and they are not selected by players.

**Comment by Armhulen** [comment]
They can be nonsensical and explained in the universe these are just sentient hud elements come on dude

**Comment by DaxDupont** [comment]
tell me the explanation of how vending machines magically grow human eyes or stickmen exist.

**Comment by Armhulen** [comment]
Stick men are animated by a crazy art wizard.

Vending machine sentience is from a virus in the event announcement. Googly eyes are from the animation staff overlay

Boom easy

**Comment by Armhulen** [comment]
And yeah if you kill the art wizard you can use his robes to create your own stick men. Completely ic.

**Comment by AlexTheSysop42** [comment]
>Googly eyes are from the animation staff overlay

yeah 10/10 IC explanation there

**Comment by DaxDupont** [comment]
Moodlets are the manifestation of people's moods and emotions like the warp gods are in 40k boom IC explanation.

**Comment by DaxDupont** [comment]
It's magic! Warp magic!

**Comment by DaxDupont** [comment]
Skeletons magic?
Technology? Magic!

**Comment by Armhulen** [comment]
then make it a creature not copying the hud.

**Comment by DaxDupont** [comment]
It's a creature. It's how you imagine your emotions, just like on your hud.

**Comment by DaxDupont** [comment]
Think of it, the sprite is literally something someone viewed as the manifestation of their emotional states.

**Comment by DaxDupont** [comment]
Maybe they are created by art wizards, surely art wizards know how to sprite. Magic is afterall how things like festivus poles live!

**Comment by PKPenguin321** [comment]
>it's either a gold spawn mob that makes absolutely no sense in its existence or another admins toy that will never be used ((SHAMEBRERO))

hello, migo?

**Comment by DaxDupont** [comment]
The great art wizard Qustinnus The Buttwizard

**Comment by Armhulen** [comment]
@PKPenguin321 but i did actually build a loreful reason for them to exist, i put creatures which had no reason to exist under netherworld and tied them with migos, spawners and the blank bodies

bad reason

**Comment by Qbopper** [comment]
this seems like the kind of pr that everyone thinks is really funny beforehand but then it gets in and its kinda just a lame joke

also i'm not really sure if making a hud element a literal thing in the game is a good idea when we already have issues with ic/ooc

**Comment by ExcessiveUseOfCobblestone** [comment]
What if the huds are based around the mob instead of the mob based around the hud? This is just the result of OOC chrono legionaries

**Comment by DaxDupont** [comment]
Closed till I or someone else implements the hud detaching first

**Pull request closed without merging**
